src/sentry/api/event_search.py

```diff
@@ -239,10 +239,6 @@ class SearchVisitor(NodeVisitor):
             "project_id",
             "project.id",
             "issue.id",
-            "device.battery_level",
-            "device.charging",
-            "device.online",
-            "device.simulator",
```

Comment:
I wonder if this is going to trip customers up 🤔

Reply:
How so?

Reply:
Values that were previously numbers are now strings, and they can't use these fields in numeric aggregations. It is overall unlikely though.